tooling/theme-editor/src/tokens/foundations/shadow/FlatShadow.tsx

```diff
@@ -29,10 +25,11 @@ export const FlatShadow = (props: FlatShadowProps): ReactElement => {
   const patternParts = getShadowParts(props.shadowPattern);
 
   return (
-    <AccordionSection
+    <Accordion
       key={`${props.themeName}-${props.innerPattern}-accordion`}
```

Comment:
why not spreading props above? (same for all the shadows)

Reply:
I just copied what was there 😄 

I didn't want to go into a rabbit hole of refactoring the theme editor code 🥲